corehq/apps/analytics/tasks.py

```diff
@@ -261,6 +267,14 @@ def track_clicked_deploy_on_hubspot(webuser, cookies, meta):
 
 
 @analytics_task()
+def job_candidate_hubspot_update(user_email):
```

Comment:
to keep with existing naming conventions this should probably be `track_job_candidate_on_hubspot`